src/elements/embed/EmbedSpec.tsx

```diff
@@ -62,15 +67,18 @@ export const createEmbedElement = (props: MainEmbedProps) =>
   createReactElementSpec(
     createEmbedFields(props),
     ({ fields, errors, fieldValues }) => {
-      return (
-        <EmbedElementForm
-          fields={fields}
-          errors={errors}
-          fieldValues={fieldValues}
-          checkThirdPartyTracking={props.checkThirdPartyTracking}
-          convertYouTube={props.convertYouTube}
-          convertTwitter={props.convertTwitter}
-        />
-      );
+      if (isCallout(fieldValues.html)) {
+        return <Callout fieldValues={fieldValues} />;
+      } else
+        return (
+          <EmbedForm
+            fields={fields}
+            errors={errors}
+            fieldValues={fieldValues}
+            checkThirdPartyTracking={props.checkThirdPartyTracking}
+            convertYouTube={props.convertYouTube}
+            convertTwitter={props.convertTwitter}
+          />
+        );
```

Comment:
I think this looks slightly nicer as ternary but makes no real difference:

```Typescript
({ fields, errors, fieldValues }) =>
      isCallout(fieldValues.html) ? (
        <Callout fieldValues={fieldValues} />
      ) : (
        <EmbedForm
          fields={fields}
          errors={errors}
          fieldValues={fieldValues}
          checkThirdPartyTracking={props.checkThirdPartyTracking}
          convertYouTube={props.convertYouTube}
          convertTwitter={props.convertTwitter}
        />
      )
```

Reply:
I think so too - have done so now.